docker/docker-compose.dev.yml

```diff
@@ -8,9 +8,8 @@ services:
       - ..:/app
     ports:
       - 3000:3000
-    environment:
-      MEILI_ADDR: http://meilisearch:7700
-      DATA_DIR: /data
+    env_file:
+      - .env  # Automatically load all variables from this file
```

Comment:
If you want to add a `.env`, let's add a `.env.sample` (even if it's empty) in the directory so that people can get it up and running quickly.